token/program-2022-test/tests/confidential_transfer.rs

```diff
@@ -2991,6 +2990,17 @@ async fn confidential_transfer_transfer_with_fee_and_split_proof_context_in_para
         close_split_context_state_accounts: None,
     };
 
+    let equality_and_ciphertext_signers = vec![
```

Comment:
Very minor: since we refer to the second type of proof as `ciphertext_validity` proof throughout the program, let's use that for consistency.
```suggestion
    let equality_and_ciphertext_proof_signers = vec![
```